edgelet/edgelet-settings/src/docker/config.rs

```diff
@@ -5,7 +5,7 @@
 pub struct DockerConfig {
     image: String,
 
-    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(skip_serializing_if = "Option::is_none", rename = "imageHash")]
```

Comment:
Do you need this rename? Doesn't rename_all (line 4) do this?

Reply:
yep, undooing